Clarify basic users and quickstart dashboards (#12998)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

  <tbody>
    <tr>
      <td>

>         Basic users are free. Basic users can set up our observability tools, run queries on their data, use custom <InlinePopover type="dashboards" /> (quickstart dashboards up to 7 days), use some basic alerting features, and more. They **can't** use our curated experiences (for example, our <InlinePopover type="apm" /> UI, browser UI, or mobile UI).

Author: ```suggestion
        Basic users are free. Basic users can set up our observability tools, run queries on your data, use custom <InlinePopover type="dashboards" /> (quickstart dashboards up to 7 days), use some basic alerting features, and more. Basic users **can't** use our curated experiences (for example, our <InlinePopover type="apm" /> UI, browser UI, or mobile UI).
```
